app/views/users/_user_list.html.erb

```diff
@@ -6,7 +6,7 @@
                 <%= presenter.gravatar_image(size: 80,
                                              class: 'img-rounded media-object pull-left') %>
                 <div class="media-body index-summary">
-                  <p class="media-heading"><%= presenter.display_name %>
+                  <p class="media-heading" title="<%= presenter.display_name unless presenter.display_name.length < 15 %>"><%= truncate(presenter.display_name, length: 14) %>
```

Comment:
@tansaku it's marginal but I take your point....how much logic is too much logic...a debate for another day ;)

There are several options here:
- I can always display the title, which gives more consistency to the experience (at the expense of a larger download)
- I can move the title thing to a helper, but then the actual display name truncation still happens separately so...another helper?  
- I could have a single helper for the entire <p class=.....?>, including the html

Also by "helper" do you mean in a separate helper file or on the User class itself?  If the latter then something like presenter.short_display_name?